requirements.txt

```diff
@@ -1,6 +1,7 @@
 numpy
 flair>=0.9
 gensim>=4.0
+scipy<1.13
```

Comment:
This issue is resulting from a dowstream library we use `gensim`. They are still in the process of releasing a fix for this, until then our best bet is to fix the scipy version. https://github.com/piskvorky/gensim/issues/3525

My PR is still WIP, for some reason when I create a PR, even if I don't assign a reviewer, I guess it sends an email to you. I will get this done most probably by tomorrow and it will be casual approval for now. Thanks for the review.

Reply:
ah I see, thanks!